### PR TITLE
[sc-2653] search by label on click

### PR DESCRIPTION
**Pull request by mpellerin42**
-  Fix Labels look & feel + use label component everywhere we display labels (row, metadata and paragraphs)
-  When clicking on a label:
    - close the viewer
    - replace search query by `LABEL={labelsetId/label}` and trigger the search
- Ignore `LABEL={…}` in suggestions

### CODE REVIEW

File: apps/search-widget-demo/src/App.svelte [hunk 1]
```diff
@@ -70,9 +70,11 @@
...
+      knowledgebox="f67d94ee-bd5b-4044-8844-a291c2ac244c"
+      apikey="eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6InNhIn0.eyJpc3MiOiJodHRwczovL3N0YXNoaWZ5LmNsb3VkLyIsImV4cCI6MTY5NjU4MTY4MiwiaWF0IjoxNjY1MDQ1NjgyLCJzdWIiOiI1MTM5ZGY1YS1lYjIxLTQ2MGItOTgzZi1hOWM4YTJiN2UwNmYiLCJqdGkiOiJhNDY0MTU3Yi05MTQyLTQxMmEtYjc2Yy1hY2EwNDhjMWY3NWIiLCJrZXkiOiI1NWE4NDNmMy03ZTU2LTRjMjYtYWU3Yi1mNGI2YjM2OTdkMGEiLCJraWQiOiIwNDA4MzFjNC04ODlmLTQ0M2EtOTI1Zi03MDI2Y2NjNGRiZTcifQ.qUDdb-a4Lv2t6ruO48x9YQqLFgUa8JeUXz5AxDmhz09ut6BGfY-rx7C5QnUcXJx6LWRFhQ5g3tTkJCRg7wrMsq4zehTyJZo4o6JdYnnVUcHgsmsh3h563ojQ8tEyn6xedzdvy5j84hXnhuFhb_k_BWwUzXUGZWEw5koBbROsrUEdudStWhdN8d1NdXGpTJ2uq-dmkofA3iVnSZNZerCQRCim3JGraUDzidvTQW1wyzBm7b1-CnJEJy9F0kjc6nNm1FA5r0958HF-Jv9PpIT0z7YcOVOayVHUc58iSEQjbubXoRXo0vycuYzSUXqs8sJLxgZYTVCRs8Y3FoLReHoYo7nnlupaKujrcZujO4vROOsZL6hGCmZdXibQ3Qc2O3RJaEEtzy7iL-uf9gexw47DymHk7erqCxB_5HQDuh9nAJBgZWK4mr3nfOxmTkAyG5c0sooUogunSYfR5xwq9KD_aSmKGqGzaNWUTJb7uFhbFRpnYczGr5lhoToQKp9T3W7_Psc_TiUeRiLMYVDNFSjM77N7AxcJ0wDx9We1F_qmex8_7Z7-9FJmRrOKi4TQrITEo-JZq59OBE314GEDGGmAoN92yjm9appmRo4w2PLTGmWc2ZxKHdWgvU-Vj4YKPl9Nl6a7CnO4mrM__tsQbLK0ZwToZF60Hq6-sPC6e5Jjq-c"
```
Comment (ebrehault) [inline apps/search-widget-demo/src/App.svelte:74]: contributor api keys in a public repo is not ideal

File: libs/search-widget/src/common/label/Label.svelte [hunk 2]
```diff
@@ -1,27 +1,50 @@
...
+  import { tap } from 'rxjs/operators';
 
   export let label: Classification;
+  export let clickable = false;
   export let removable = false;
 
+  let setColor = true;
   const color = labelSets.pipe(
     map((labelSet) => labelSet[label.labelset]?.color),
+    tap((color: string) => {
```
Comment (ebrehault) [inline libs/search-widget/src/common/label/Label.svelte:16]: It is not obvious why we should do that.
Comment maybe?

File: libs/search-widget/src/common/label/Label.svelte [hunk 3]
```diff
@@ -1,27 +1,50 @@
...
+
   const dispatch = createEventDispatcher();
   const remove = () => {
     dispatch('remove');
   };
+  const onClick = (event: MouseEvent | KeyboardEvent) => {
+    if (clickable) {
+      event.preventDefault();
+      event.stopPropagation();
+      dispatch('selected');
```
Comment (ebrehault) [inline libs/search-widget/src/common/label/Label.svelte:33]: any reason why you prefer `selected` over `click`?

**Review by ebrehault — APPROVED** [review]

File: libs/search-widget/src/common/label/Label.svelte (hunk 3 above)
Comment (mpellerin42) [reply libs/search-widget/src/common/label/Label.svelte:33]: I thought using click would not allow me to trigger the click even only when clickable was enabled

**Review by ebrehault — APPROVED** [review]

**Review by ebrehault — APPROVED** [review]

**Pull request merged**